README.md

```diff
@@ -32,250 +32,16 @@ X_n &\dist \distNorm(\hmu, \hSigma)
 The best way to use ShorTeX is to download a copy of `shortex.sty` and put it
 in your document's root folder. Then in your document's preamble, type:
 ```latex
-\usepackage[autonum]{shortex}
+\usepackage{shortex}
 ```
 **Note: ShorTeX requires you to compile your document 4 times.**
 
-Do not install ShorTeX system-wide (yet). This repository has not reached
-a stable v1.0, and we are updating things regularly without any guarantee of 
+Do not install ShorTeX system-wide. This repository has not reached
+a stable v1.0, and we updating things regularly without any guarantee of 
```

Comment:
typo: "we updating" => "we are updating"

Reply:
done